fastchat.md

```diff
@@ -0,0 +1,29 @@
+## Benchmarking huggingface models via FastChat 
+
+FastChat supports a range of huggingface models and provide serving APIs. LLMeBench has an interface to FastChat APIs allowing to query and get responses from models.
+Here is a quick start to load a huggingface model with FastChat and run an asset from LLMeBench on the model. 
+### Install FastChat
+
+```bash
+pip3 install "fschat[model_worker,webui]"
+```
+
+## Serving models with Web GUI
+
+#### Launch the controller
+```bash
+python3 -m fastchat.serve.controller
+```
+
+#### Launch the model worker(s)
+```bash
+python3 -m fastchat.serve.model_worker --model-path gpt2 --host localhost --port 5004
+```
+Wait until the process finishes loading the model and you see "Uvicorn running on ...". The model worker will register itself to the controller .
+
+
+#### Run the asset from LLMeBench
```

Comment:
@bsabri Can we add an asset specifically designed for gpt2? I assume the gpt2 adapter does not really care about the "system" role? 
```python
[
        {
            "role": "system",
            "content": "You are an expert translator specialized in translating texts from Arabic to English. You are concise as you only output the translation of the text without any illustrations or extra details",
        },
        {
            "role": "user",
            "content": f"Translate the following text to English.\nText: {input_sample}\nTranslation: ",
        },
    ]
```

We can either add the asset properly or just show the prompt function here in the doc (perhaps for a task that gpt2 is actually good/decent at)